cmd/sdk-server/Dockerfile

```diff
@@ -21,4 +21,5 @@ COPY --chown=agones:root ./bin/sdk-server.linux.amd64 /home/agones/sdk-server
 COPY --chown=agones:root ./bin/LICENSES ./bin/dependencies-src.tgz /home/agones/
 
 USER agones
-ENTRYPOINT ["/home/agones/sdk-server"]
\ No newline at end of file
+ENTRYPOINT ["/home/agones/sdk-server"]
+CMD [ "--grpc-port", "59357" ]
```

Comment:
Does it work without this change? The binary already defaults this flag (in code) so it doesn't seem like we should also need to default it in the dockerfile.

Reply:
Let me check your assumption.

Reply:
No need for this `CMD` block, removed